modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro

```diff
@@ -18,6 +28,26 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro createDataSetView {
```

Comment:
This macro has been modified in abhner's original PR, can you take a look there?

Reply:
Yes, I'll do it